gapic/src/main/com/google/gapid/server/ChildProcess.java

```diff
@@ -104,12 +105,24 @@ protected void runProcess(final SettableFuture<T> result, final ProcessBuilder p
     } catch (InterruptedException e) {
       LOG.log(INFO, "Killing " + name);
       result.setException(e);
-      process.destroy();
+      killProcess();
     } finally {
       onExit(exitCode);
     }
   }
 
+  protected void killProcess() {
+    process.destroy();
+    try {
+      boolean destroyed = process.waitFor(500, TimeUnit.MILLISECONDS);
```

Comment:
This seems arbitrary and way to short.

Reply:
My stance is that there may be a lot of reasons why GAPIS does not terminate, and we need to be robust to this. So if GAPIS does not terminate within a certain duration after process.Destroy(), we force-kill it.

The duration is completely arbitrary, indeed. How long would you allow GAPIS to terminate properly? 2 seconds?